app/assets/stylesheets/master.scss

```diff
@@ -19,9 +19,6 @@
 // Themes for each college
 @import "themes/*";
 
-// Omni-Nav styles
-@import "omni_nav/helpers/mixins-and-variables.scss";
-// @import "helpers/icomoon.scss";
-@import "omni_nav/modules/etc.scss";
-@import "omni_nav/modules/omni-nav.scss";
-@import "omni_nav/modules/off-canvas-nav.scss";
+//Omni-nav styles
+@import "omni_nav/**/*";
```

Comment:
I prefer this. But just to be sure: this isn't going to pull in any dead files that should not be pulled in, is it?